ark/utils/plot_utils.py

```diff
@@ -28,52 +68,21 @@ def plot_overlay(predicted_contour, plotting_tif, alternate_contour=None, path=N
             2D or 3D numpy array of imaging signal
         alternate_contour (numpy.ndarray):
             2D numpy array of labeled cell objects
-        path (str):
-            path to save the resulting image
-        show (bool):
-            whether or not to show plot
     """
 
+    # just plot the outlines
     if plotting_tif is None:
-        # will just plot the outlines
         pass
     else:
-        if len(plotting_tif.shape) == 2:
-            if plotting_tif.shape != predicted_contour.shape:
-                raise ValueError("plotting_tif and predicted_contour array dimensions not equal.")
-            else:
-                # convert RGB image with same data across all three channels
-                plotting_tif = np.stack((plotting_tif, plotting_tif, plotting_tif), axis=2)
-        elif len(plotting_tif.shape) == 3:
-            blank_channel = np.zeros(plotting_tif.shape[:2] + (1,), dtype=plotting_tif.dtype)
-            if plotting_tif.shape[2] == 1:
-                # pad two empty channels
-                plotting_tif = np.concatenate((plotting_tif, blank_channel, blank_channel), axis=2)
-            elif plotting_tif.shape[2] == 2:
-                # pad one empty channel
-                plotting_tif = np.concatenate((plotting_tif, blank_channel), axis=2)
-            elif plotting_tif.shape[2] == 3:
-                # don't need to do anything
-                pass
-            else:
-                raise ValueError("only 3 channels of overlay supported, got {}".
-                                 format(plotting_tif.shape))
-        else:
-            raise ValueError("plotting tif must be 2D or 3D array, got {}".
-                             format(plotting_tif.shape))
-
-    if path is not None:
-        if os.path.exists(os.path.split(path)[0]) is False:
-            raise ValueError("File path does not exist.")
+        plotting_tif = preprocess_tif(predicted_contour, plotting_tif)
 
     # define borders of cells in mask
     predicted_contour_mask = find_boundaries(predicted_contour,
                                              connectivity=1, mode='inner').astype(np.uint8)
     predicted_contour_mask[predicted_contour_mask > 0] = 255
 
     if plotting_tif is None:
-        # will just save the contour mask
-        io.imsave(path, predicted_contour_mask)
+        return predicted_contour_mask
```

Comment:
We should move this functionality (creating just the outlines) to `save_segmentation_labels`, since we'll want to do this for every FOV